examples/Azure_IoT_Adu_ESP32/Azure_IoT_Adu_ESP32.ino

```diff
@@ -63,7 +63,7 @@
 #define SAMPLE_MQTT_PAYLOAD_LENGTH 1024
 
 // ADU Values
-#define ADU_PPV_DTMI "dtmi:azure:iot:deviceUpdateContractModel;1"
+#define ADU_PPV_DTMI "dtmi:azure:iot:deviceUpdateContractModel;2"
```

Comment:
This can be removed and the value from the header file can be used instead.

`AZ_IOT_ADU_CLIENT_AGENT_MODEL_ID`

Reply:
We need documentation later to explain what does this mean to the customers, and how they can change it as soon as it has the deviceUpdateContractModel as a pnp subcomponent. Something for later.

Reply:
Change done, as you suggested.